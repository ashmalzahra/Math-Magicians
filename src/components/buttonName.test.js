import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Button from './buttonName';

describe('Checking page deployment', () => {
  test('Button component matches snapshot', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders Button component', () => {
    const onClick = () => {};

    render(<Button name="AC" className="grey" onClick={onClick} />);

    expect(screen.getByRole('button')).toBeInTheDocument();
    expect(screen.getByText(/AC/)).toBeInTheDocument();
  });
});