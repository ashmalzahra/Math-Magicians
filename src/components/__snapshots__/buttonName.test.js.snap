// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking page deployment Button component matches snapshot 1`] = `
<button
  type="button"
/>
`;
